docs/execution_providers/DirectML-ExecutionProvider.md

```diff
@@ -0,0 +1,97 @@
+# DirectML Execution Provider (Preview)
```

Comment:
Do we want to document the fact that parallel execution mode and mem pattern should be disabled before when using DML?

Reply:
Ah, yes, that's a good point! I'll add that now.